integration/yarn/src/test/java/tachyon/yarn/ApplicationMasterTest.java

```diff
@@ -126,7 +127,15 @@
 
   @Before
   public void before() throws Exception {
-    mMaster = new ApplicationMaster(NUM_WORKERS, MASTER_ADDRESS, RESOURCE_ADDRESS);
+    setupApplicationMaster(ImmutableMap.<String, String>of());
```

Comment:
A new `TachyonConf` is constructed locally during the test and passed into the `ApplicationMaster` constructor, so there's no state to reset.